### PR TITLE
Update for the new onLeftArrow, onRightArrow in draft-js 0.10.4

### PR DESCRIPTION
**Pull request by Dartoxian**
https://github.com/facebook/draft-js/releases has a new release, v0.10.4 which adds onLeftArrow and onRightArrow to Editor props.

### CODE REVIEW

**Comment by Dartoxian** [comment]
Hey @willisplummer would it be possible to take a look at this?

**Review by willisplummer — APPROVED** [review]
👍 lgtm

**Comment by RyanCavanaugh** [comment]
Approved by a listed owner. PR ready to merge pending express review by a maintainer.

**Pull request merged**
